modules/i18n/src/translations/en-US/portals/claims.ts

```diff
@@ -386,6 +386,7 @@ export const claims: ClaimsNS = {
                 title: {
                     dialect: "Add an Attribute Mapping",
                     external: "Add an {{type}} Attribute",
+                    externalReadOnly: "No {{type}} Attributes Added",
```

Comment:
let's use a key that better reflects the meaning of the text.

Reply:
Fixed with [4a16748](https://github.com/wso2/identity-apps/pull/8729/commits/4a16748ffa9f0aa11018c72d61dbda1b72cbd546).